KUBERNETES_CSI_OWNERS_ALIASES

```diff
@@ -0,0 +1,43 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+aliases:
+
+  # SIG-Storage chairs and leads should always have approval rights in all repos.
+  # Others may be added as needed here or in each repo.
+  kubernetes-csi-approvers:
+  - jsafrane
+  - msau42
+  - saad-ali
+  - xing-yang
+
+  # Reviewers are automatically assigned to new PRs. The following
+  # reviewers will be active in all repos. Other reviewers can be
+  # added in each repo.
+  #
+  # Reviewers are encouraged to set the "Busy" flag in their GitHub status
+  # when they are temporarily unable to review PRs.
+  kubernetes-csi-reviewers:
+  - andyzhangx
+  - chrishenzie
+  - huffmanca
+  - j-griffith
+  - jingxu97
+  - jsafrane
+  - msau42
```

Comment:
@msau42: does your workload allow keeping you in the list of default reviewers?

Reply:
You can remove me from reviewers. I think we have a solid reviewer pool now :-)

Reply:
Done.